diff -r -u /var/aegir/platforms/drupal-7.31-porto1c/sites/all/modules/remote_stream_wrapper/remote_stream_wrapper.module ./remote_stream_wrapper.module
--- /var/aegir/platforms/drupal-7.31-porto1c/sites/all/modules/remote_stream_wrapper/remote_stream_wrapper.module	2014-09-22 06:17:38.000000000 +0200
+++ ./remote_stream_wrapper.module	2014-10-02 00:01:49.000000000 +0200
@@ -143,7 +143,7 @@
  * Validation callback for remote URLs.
  */
 function remote_stream_wrapper_validate_url($element, &$form_state) {
-  $value = trim($element['#value']);
+  $value = $element['#value'] = str_replace(" ","%20",trim($element['#value']));
   if ($value != '' && !valid_url($value, TRUE)) {
     form_error($element, t('Invalid URL %url.', array('%url' => $value)));
   }
@@ -187,7 +187,7 @@
   $file = new stdClass();
   $file->fid = NULL;
   $file->uri = $uri;
-  $file->filename = basename($file->uri);
+  $file->filename = urldecode(basename($file->uri));
   $file->filemime = file_get_mimetype($file->uri);
   $file->uid = $GLOBALS['user']->uid;
   $file->status = FILE_STATUS_PERMANENT;
